CMSgov/eAPD
# Configs for connecting to LaunchDarkly

### Description
Added code to connect to LaunchDarkly on the back and frontend. Also added some code to dispatch flag changes throughout the system so that they can be used in redux.

### Significant changes or possible side effects
I wrapped the LaunchDarkly code around Root. That shouldn't cause any issues, but just make sure you aren't seeing weird stuff in the console.

I also have some code as proof of concepts in here, so I will be removing that before or after merging, I guess.

### Feature Flags
validation: true

### Steps to manually verify this change

1. Login to eAPD
2. Create a new eAPD
3. Open the eAPD
4. On the APD Overview you should see the message VALIDATION ON at the top of the page, the fields should show errors, and in the network console, you should see that { validation: true } is being returned as part of the APDs.
5. If you have access to the flags in LaunchDarkly, you can turn the validation flag off and confirm that none of the above appears.

VALIDATION ON is verification that the react hooks are connected to LaunchDarkly on the frontend
validation errors show that the flag values are getting to redux
validation: true being returned with the apd values shows that LaunchDarkly is working in the backend

### This pull request is ready to code review when

      tests, refactor

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
